gui.go

```diff
@@ -634,3 +642,44 @@ func (g *Gui) execKeybindings(v *View, ev *termbox.Event) (matched bool, err err
 	}
 	return matched, nil
 }
+
+func (g *Gui) getTermSize() (int, int, error) {
+	var sz struct {
+		rows uint16
+		cols uint16
+	}
+
+	var termw, termh int
+
+	out, err := os.OpenFile("/dev/tty", os.O_RDWR, 0)
```

Comment:
This might prevent gocui from working under systems that are not linux (correct me if im wrong)

Reply:
Also if this check is just for checking if it's ran inside docker this will then never work in a native windows docker container.

Reply:
Docker container as the Linux container so we just need to check and get the term window for Linux.
However gocui support windows so There need to check runtime os.
I’ll fixe this logic.

Reply:
Also i'm not sure how much people use this library on BSD based OSes but it might be handy to support those too.

Reply:
I fixed this logic.